lib/rollup.config.js

```diff
@@ -6,6 +6,30 @@ import pkg from "./package.json";
 import commonPkg from "../package.json";
 
 const components = require("./src/index.json");
+
+function injectCSS(css, uniqueId, isPrepend) {
+    if (!css || !document) {
```

Comment:
is the suggestion from Aaron no longer needed?

Reply:
yeah it's still needed (and other stuff on develop) just making sure the history is back to what master used to be before I merge in the new changes haha